### PR TITLE
Raise an informative error when input data have NAs (also update geosnap links)

### PR DESCRIPTION
**Pull request by renanxcortes**
This implements https://github.com/pysal/segregation/pull/130#issuecomment-522137184. So, this is PR is open in favor of https://github.com/pysal/segregation/pull/130.

### CODE REVIEW

**Comment by knaaptime** [comment]
should we go ahead and close #130?

File: segregation/aspatial/aspatial_indexes.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 from scipy.stats import norm
 from scipy.optimize import minimize
 
-from segregation.util.util import _dep_message, DeprecationHelper
+from segregation.util.util import _dep_message, DeprecationHelper, _nan_handle
```
Comment (knaaptime) [inline segregation/aspatial/aspatial_indexes.py]: we missed an opportunity here to call this "nandle" 😝

**Comment by renanxcortes** [comment]
> should we go ahead and close #130?

I'm +1 for that.

**Comment by renanxcortes** [comment]
> > should we go ahead and close #130?
> 
> I'm +1 for that.

But before, I'd like to check if this one is okay merging for you.

**Comment by knaaptime** [comment]
as i keep thinking about this, I think a warning is most appropriate. It's not really an error of any kind, so there's no need to fail just because NAs are present, but we should let the user know

**Comment by renanxcortes** [comment]
> as i keep thinking about this, I think a warning is most appropriate. It's not really an error of any kind, so there's no need to fail just because NAs are present, but we should let the user know

Sure, just changed this in https://github.com/pysal/segregation/pull/131/commits/cb5182a604e0bf0f358c3d2ecd493ef7dda5661b.

**Pull request merged**